cmake/celerity-config.cmake.in

```diff
@@ -50,6 +50,7 @@ set(CELERITY_FEATURE_SCALAR_REDUCTIONS "@CELERITY_FEATURE_SCALAR_REDUCTIONS@")
 set(CELERITY_FEATURE_SIMPLE_SCALAR_REDUCTIONS "@CELERITY_FEATURE_SIMPLE_SCALAR_REDUCTIONS@")
 set(CELERITY_FEATURE_LOCAL_ACCESSOR "@CELERITY_FEATURE_LOCAL_ACCESSOR@")
 set(CELERITY_FEATURE_UNNAMED_KERNELS "@CELERITY_FEATURE_UNNAMED_KERNELS@")
+set(CELERITY_DETAIL_HAS_NAMED_THREADS "@CELERITY_DETAIL_HAS_NAMED_THREADS@")
```

Comment:
For `_DETAIL_` flags the CMake variable shouldn't be included in the installed package; the preprocessor definition will still be set on anything that links to the runtime library.

Reply:
I wasn't sure whether to include it here or not. After discussing it with @fknorr I tentatively settled on including it. But if you say it's not necessary I will remove it.

Reply:
Done.